sliceguard/data.py

```diff
@@ -15,43 +21,97 @@ def _get_tutorial_imports():
     return downloader
 
 
-def from_huggingface(dataset_identifier: str):
+def write_tempfile(data: dict):
```

Comment:
@tarekt  This will not work! Files will be deleted instantly after context is closed. Also they need a file ending. Consider using a persistent directory which the user specifies.